src/shared/components/SharePost.tsx

```diff
@@ -39,8 +44,19 @@ const SharePost = ({ profilePic, theme }: SharePostProps) => {
     const res = await axios.post(endPoint, postData, {
       withCredentials: true,
     });
+    const newPostInfo = {
+      ...res.data.data,
+      user: {
+        firstName: user.firstName,
+        lastName: user.lastName,
+        profilePic: user.profilePic,
+      },
+      likes: [],
+      comments: [],
+    };
```

Comment:
You might want to build a data type for this since we probably need to reuse it

Reply:
We have a type in the types folder for it. The BaseFeedType should be reusable for the profile page.
![image](https://user-images.githubusercontent.com/91350990/213265338-6a1617c8-8ca3-490c-aaa2-f1a7bd09c4c7.png)

We structured it similar to what was being sent back from the [GET] /feed endpoint.